src/Block/BreadcrumbBlockService.php

```diff
@@ -83,7 +83,7 @@ protected function getMenu(BlockContextInterface $blockContext): ItemInterface
 
     private function getName(): string
     {
-        return 'sonata.page.block.breadcrumb';
+        return 'page';
```

Comment:
IMO this is not the correct change. The metadata can return the previous value and the handleContext can check against page, and we can remove this method.

Reply:
I reverted.

I didn't follow this dicussion so prefer to let @eerison doing another PR about it.

Reply:
well but this values need to be equals passed by template!

if we use something different of `page`, then we still have the same problem to use `sonata.page.block.breadcrumb`

what is your suggestion for this @jordisala1991 ?